api/v1_projects.inc

```diff
@@ -438,7 +444,9 @@ function api_v1_project_pagedetails($method, $data, $query_params)
                 $round_data["last_modified"] = date(DATE_ATOM, $round_data["modified_timestamp"]);
                 unset($round_data["modified_timestamp"]);
             }
-            unset($round_data["username"]);
+            if (!can_user_see_usernames_for_page($project, $image, $rounds_to_display)) {
```

Comment:
If we're solving this here, we should make the solution work for `api_v1_project_page_round()` too, eg:
```php
    // We can't show the username here unless the user has proofed the page
    // in one round or they are an PF/SA. We need to abstract the conditional
    // logic out of tools/project_manager/page_detail.inc
    unset($row["user"]);
```

Reply:
Done.